pgjdbc/src/main/java/org/postgresql/util/PGtokenizer.java

```diff
@@ -20,6 +23,15 @@
  * @see org.postgresql.geometric.PGpolygon
  */
 public class PGtokenizer {
+  
+	static final Map<Character, Character> closing2OpeningCharacter = new HashMap<>();
```

Comment:
private
since this is a constant, put name in all caps.
initialize to appropriate size (6)